vulcan/lib/server/workflows/scripts/merge_anndata_from_raw_h5.py

```diff
@@ -0,0 +1,30 @@
+from archimedes.functions.dataflow import output_path, input_path
+from archimedes.functions.metis import metis_file_path
+from archimedes import scanpy
+
+magma_output = input_tsv('magma_output')
```

Comment:
Do you need to import `input_tsv` from `archimedes.functions.dataflow`?

Reply:
yup.  And likely need to add `.functions.scanpy` in line 3. This script is untested (only `magma_retrieve_paths.py` has been) and has some pseudo-code lines remaining.